spec/jobs/google_scraping_job_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleScrapingJob, type: :job do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    it 'enqueues google scraping job' do
+      user = Fabricate(:user)
+      keyword = Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      expect do
+        GoogleScrapingJob.perform_later(keyword.id, keyword.keyword)
+      end.to have_enqueued_job(GoogleScrapingJob)
+
+      assert_enqueued_with(job: GoogleScrapingJob, args: [keyword.id, 'AWS'])
```

Comment:
not sure we need the `assert_enqueued_with` here as the `have_enqueued_job` has the `with` option too

https://relishapp.com/rspec/rspec-rails/docs/matchers/have-enqueued-job-matcher

Reply:
Fixed in 5296c517e903326287635519b47c35e009c5d932